### DISCUSSION TITLE
Editor Slideouts sidebar visibility toggle at large up

### DISCUSSION THREAD

**Discussion by johnwbaxter** [craftcms/cms#10349] [Ideas]
The Editor slideout is great, i love it, but I think it's main purpose (for me anyway) is to edit content and certainly for me, 99% of the time, it's the page content, not the entry type or post date etc.

So, it feels odd that the sidebar in the slide out, which has the entrytype/slug/post date etc, takes up 50% of the slideout space and there is no way to change that at desktop view. However, when we come down to more mobile type widths, you can toggle it's visibility.

So, It would be great if that toggle button was there 100% of the time so It can be moved out of sight so we can use the full space to edit content. Please!

Thanks y'all.

:)

**Comment by brandonkelly** [comment]
It’s only always-visible for larger screen sizes. Will Consider an option to hide though.
